build/Nuget/Microsoft.VisualStudio.ProjectSystem.nuspec

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.VisualStudio.ProjectSystem</id>
+    <summary>CoreXT package for Microsoft Project System for C# and VB </summary>
+    <description>
+      This Project system is a re-write of the C# and VB project system in Managed Language
+    </description>  
+    <language>en-US</language>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <!-- Since VS is the only consumer, now, the version is set to 1.0.0-->
+    <version>1.0.0</version>
```

Comment:
Isn't this version just overwritten? ie 1.0.0 is meaningless because you pass the version above, right?